src/SIL.XForge.Scripture/ClientApp/src/app/app.component.scss

```diff
@@ -36,6 +36,9 @@
       // KHSV05 is one example that will cause overflow, which looks particularly bad in RTL mode because it
       // overflows to the left in a menu that is layed out in LTR.
       overflow-wrap: anywhere;
+
+      height: unset;
+      min-height: 48px;
```

Comment:
This is just restoring some CSS that was removed by mistake (it was right beside some rules that were no longer needed).

MDC sets the height to 48px. We want it to be a minimum of 48px, but we want the content to allow it to grow larger.